ModularTegustation/fishing/code/turfs.dm

```diff
@@ -19,12 +19,15 @@
 	var/sound_delay = 0
 	//Lootlist of things for fishing.
 	var/list/environment = list(/obj/item/food/grown/harebell = 200)
-	var/list/static/forbidden_types = typecacheof(list(
+	//Things that just cant sink. Dont bother trying to sink them.
+	var/static/list/cant_sink_types = typecacheof(list(
```

Comment:
I believe `/effect` in general should be added to this, as opposed to various different subtypes.

Reply:
Got it